test/functional/rpc_createmultisig.py

```diff
@@ -72,6 +75,14 @@ def run_test(self):
             assert_equal(legacy_addr, node0.addmultisigaddress(2, keys, '', 'bech32')['address'])
             assert_equal(legacy_addr, node0.addmultisigaddress(2, keys, '', 'p2sh-segwit')['address'])
 
+        self.log.info('Testing sortedmulti descriptors with BIP 67 test vectors')
+        with open(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data/rpc_bip67.json'), encoding='utf-8') as f:
+            vectors = json.load(f)
+
+        for t in vectors:
+            key_str = ','.join(t['keys'])
+            self.nodes[0].deriveaddresses(descsum_create('sh(sortedmulti(2,' + key_str + '))'))[0] == t['address']
```

Comment:
This just returns `false` without failing the test.  In fact all these addresses are incorrect; you either have to run the node in mainnet (like `feature_config_args`), or replace the test vectors for regtest. I manually verified with a mainnet node that the vectors do pass.

Reply:
Whoops. Fixed. Changed the addresses to regtest ones. You can check they still map to the same scriptPubKey using `getaddressinfo` and `decodescript`.